### ISSUE TITLE
Could you share more detailed code? 

### ISSUE THREAD

**Issue by airsyuan** [Debapriya-Tula/Video-inbetweening#2]
Hi, thanks for sharing the part of the code, and this is a quiet interesting project. However, only a part of code is 
released. I just begin to learn keras, it's a little hard for me to complete the code.  And I would appreciate it if you can share more detailed code.

**Comment by Debapriya-Tula** [comment]
I am afraid I am no longer working on this approach. I was stuck with the discriminator loss. I couldn't understand it. Later on, I found a more suitable approach to my primary problem which was to reconstruct the frames in between, not generate as in a Generator would do, Please find my other repo viz, AdaConv-Pytorch which better suits my problem. It is based on a GitHub code by Hyeongmin Lee. I have modified the code for the Visual Tracker Benchmark dataset.

**Comment by airsyuan** [comment]
Ok. I will try it later. Thanks for your kind reply.
